beacon/blsync/config.go

```diff
@@ -72,39 +72,58 @@ var (
 )
 
 func makeChainConfig(ctx *cli.Context) lightClientConfig {
+	var (
+		config       lightClientConfig
+		predefConfig bool
+	)
+	customConfig := ctx.IsSet(utils.BeaconConfigFlag.Name)
 	utils.CheckExclusive(ctx, utils.MainnetFlag, utils.GoerliFlag, utils.SepoliaFlag)
```

Comment:
Wouldn't a simpler fix be to just extend this line, so that
```
utils.CheckExclusive(ctx, utils.MainnetFlag, utils.GoerliFlag, utils.SepoliaFlag, utils.BeaconConfigFlag.Name)
```

Reply:
I mean, you can keep the checks with "Custom beacon chain .. but genesis root is missing" but the `predefConfig` seens not needed

Reply:
Indeed, this is equivalent and simpler.